site/content/docs/user/known-issues.md

```diff
@@ -234,6 +235,19 @@ an 03 17:42:41 kind-1-control-plane kubelet[3804]: F0103 17:42:41.470269 3804 ku
 
 This problem seems to be related to a [bug in Docker][moby#9939].
 
+## Encountered permission denied
+
+When using `kind`, we assume that the user you are using has direct access to docker sokect.
```

Comment:
`sokect` is a typo, I'm also not sure if users reading this will know what the socket is. maybe "the user you are executing kind as has permission to use docker"

Reply:
Thanks. Updated.